File: app/src/main/java/ru/visualmath/android/api/model/Block.java
package ru.visualmath.android.api.model;

import com.google.gson.annotations.SerializedName;

public class Block {

    @SerializedName("ended")
    private boolean ended;

    @SerializedName("results")
    private Results results;

    public Results getResults() {
        return results;
    }

    public boolean isEnded() {
        return ended;
    }
}

File: app/src/main/java/ru/visualmath/android/api/model/Block.kt
package ru.visualmath.android.api.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Block(

        @SerializedName("ended")
        val isEnded: Boolean = false,

        @SerializedName("results")
        val results: Results = Results()
) : Parcelable
